packages/plugin-core/src/_extension.ts

```diff
@@ -735,85 +733,15 @@ export async function _activate(
-      // only need to run this for non-mac
-      if (vimInstalled && os.type() !== "Darwin") {
-        Logger.info({
-          ctx,
-          msg: "checkAndApplyVimKeybindingOverrideIfExists:pre",
-        });
-        AnalyticsUtils.track(ExtensionEvents.VimExtensionInstalled);
-        const keybindingPayload =
-          KeybindingUtils.checkAndApplyVimKeybindingOverrideIfExists();
-        if (keybindingPayload.data) {
-          const { keybindingConfigPath, newKeybindings: resolvedKeybindings } =
-            keybindingPayload.data;
-          keybindingPath = keybindingConfigPath;
-          if (!_.isUndefined(resolvedKeybindings)) {
-            const today = Time.now().toFormat("yyyy.MM.dd.HHmmssS");
-            const maybeBackupPath = `${keybindingConfigPath}.${today}.vim.old`;
-            if (!fs.existsSync(keybindingConfigPath)) {
-              fs.ensureFileSync(keybindingConfigPath);
-              fs.writeFileSync(keybindingConfigPath, "[]");
-            } else {
-              fs.copyFileSync(keybindingConfigPath, maybeBackupPath);
-              backupPaths.push(maybeBackupPath);
-            }
-            writeJSONWithComments(keybindingConfigPath, resolvedKeybindings);
-            AnalyticsUtils.track(ExtensionEvents.VimExtensionInstalled, {
-              fixApplied: true,
-            });
-          }
-        } else {
-          Logger.info({ ctx, msg: "unable to apply keybinding fix" });
-          // TODO: report error
-        }
-      }
-    }
-
-    // TODO: remove, we are not tracking this today
-    // if (extensionInstallStatus === InstallStatus.UPGRADED) {
-    //   const { keybindingConfigPath, migratedKeybindings } =
-    //     KeybindingUtils.checkAndMigrateLookupKeybindingIfExists();
-    //   keybindingPath = keybindingConfigPath;
-    //   if (!_.isUndefined(migratedKeybindings)) {
-    //     const today = Time.now().toFormat("yyyy.MM.dd.HHmmssS");
-    //     const maybeBackupPath = `${keybindingConfigPath}.${today}.lookup.old`;
-    //     fs.copyFileSync(keybindingConfigPath, maybeBackupPath);
-    //     backupPaths.push(maybeBackupPath);
-    //     writeJSONWithComments(keybindingConfigPath, migratedKeybindings);
-    //   }
-    // }
-
-    if (backupPaths.length > 0) {
-      vscode.window
-        .showInformationMessage(
-          "Conflicting or outdated keybindings have been updated. Click the button below to see changes.",
-          ...["Open changes"]
-        )
-        .then(async (selection) => {
-          if (selection) {
-            const uri = vscode.Uri.file(keybindingPath);
-            await VSCodeUtils.openFileInEditor(uri);
-            backupPaths.forEach(async (backupPath) => {
-              const backupUri = vscode.Uri.file(backupPath);
-              await VSCodeUtils.openFileInEditor(backupUri, {
-                column: vscode.ViewColumn.Beside,
-              });
-            });
-          }
-        });
-    }
-
     if (userOnOldVSCodeVer) {
```

Comment:
if you rebase, this block should now be removed

Reply:
I don't think it's removed in master right now. https://github.com/dendronhq/dendron/blob/ca708505d62927f28614ce8cd83fcc515b68496a/packages/plugin-core/src/_extension.ts#L813